packages/ember-runtime/lib/system/core_object.js

```diff
@@ -873,15 +852,13 @@ var ClassMixinProps = {
     @private
   */
   eachComputedProperty(callback, binding) {
-    var property;
-    var empty = {};
-
-    var properties = get(this, '_computedProperties');
+    this.proto(); // ensure prototype is initialized
+    let empty = {};
 
-    for (var i = 0; i < properties.length; i++) {
-      property = properties[i];
-      callback.call(binding || this, property.name, property.meta || empty);
-    }
+    let m = meta(this.prototype);
+    m.forEachDescriptor((name, descriptor) => {
```

Comment:
@krisselden - I tested this branch against crates.io (locally checked out and linked), and I confirmed that `meta.forEachDescriptor` was inlined in `applyMixins`, but I do not see `eachComputedProperty` itself listed in the irhydra output.  (I uploaded to our shared profiles folder if you want to see the results)